projects/auth0-angular/src/lib/auth.service.ts

```diff
@@ -137,7 +145,13 @@ export class AuthService<TAppState extends AppState = AppState>
   loginWithRedirect(
     options?: RedirectLoginOptions<TAppState>
   ): Observable<void> {
-    return from(this.auth0Client.loginWithRedirect(options));
+    const sub = new Subject<void>();
+    this.authClient
+      .getInstance$()
+      .pipe(mergeMap((client) => client.loginWithRedirect(options)))
+      .subscribe(sub);
+
+    return sub.asObservable();
```

Comment:
I have done this for backwards compatibility with the v1 API. I will add a follow up PR that cleans up this, but would like to dedicate a PR to it as it's a breaking change on its own.